core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -104,7 +105,7 @@ class KafkaController(val config: KafkaConfig,
   private val isrChangeNotificationHandler = new IsrChangeNotificationHandler(eventManager)
   private val logDirEventNotificationHandler = new LogDirEventNotificationHandler(eventManager)
 
-  @volatile private var activeControllerId = -1
+  @volatile private var activeControllerId = KafkaController.NO_CONTROLLER_ID
```

Comment:
If we're going to the trouble of changing this, how about we make it an `Option`?